jcommune-view/jcommune-web-controller/src/test/java/org/jtalks/jcommune/web/controller/UserControllerTest.java

```diff
@@ -119,7 +119,8 @@ public void testInitBinderDefaultStringEditor() {
     }
 
     @Test
-    public void testRegistrationPage() throws Exception {
+    public void testRegistrationPageWhenUserNotLogged() throws Exception {
```

Comment:
Let's name it in BDD style: `userMustBeAbleToOpenRegistrationPage_ifHeIsNotLoggedIn`